ext/debug/ctf/bottoms/posix/ctf_bottom.c

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
@nashif : What is the policy for a module which is partly external & MIT, like in this case? Is it sensible to place the whole code in ext/?